js/script.js

```diff
@@ -0,0 +1,28 @@
+const inputItem = document.getElementById('addItem');
+const keepList = document.querySelector('.list');
+
+function createCustomElement(tag, className = '', innerText = '') {
+  const element = document.createElement(tag);
+  element.className = className;
+  element.innerText = innerText;
+  return element;
+}
+
+function addItemElement(text) {
+  const itemElement = createCustomElement('li', 'item');
+  const textElement = createCustomElement('span', 'item-text', text);
+  const clearButton = createCustomElement('button', 'remove-item-btn', 'x');
+  clearButton.addEventListener('click', (ev) => ev.target.parentElement.remove());
+  itemElement.appendChild(textElement);
+  itemElement.appendChild(clearButton);
+  keepList.appendChild(itemElement);
+}
+
+function addItem(event) {
+  if (event.keyCode === 13) {
+    addItemElement(event.target.value);
+    event.target.value = '';
```

Comment:
Jony Lint, me ilumina nessa func por favor. O que ela está fazendo?

Reply:
A condição verifica se a tecla digitada é <kbd>Enter</kbd>, e se for, adiciona o texto digitado como um item na lista e limpa o conteúdo do `<input>`. Talvez deveria colocar um comentário no código para ficar mais claro?

EDIT: Hmm, e agora vi que é possível adicionar um item vazio na lista kkk!
```suggestion
function addItem(event) {
  // se a tecla Enter for pressionada e tiver algum conteúdo no input
  if (event.keyCode === 13 && event.target.value.trim().length > 0) {
    addItemElement(event.target.value.trim());
    event.target.value = '';
```